### PR TITLE
First implementation of the Cell class

### PR DESCRIPTION
**Pull request by manuelkoso**

### CODE REVIEW

File: src/main/java/Cell.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import org.jetbrains.annotations.NotNull;
+
+public class Cell {
+
+    public enum Status {FREE, OCCUPIED}
+
+    private Stone stone;
+    private Status status = Status.FREE;
```
Comment (peiva-git) [inline src/main/java/Cell.java:8]: Instead of Status, what about a boolean occupied field?

**Review by peiva-git — CHANGES_REQUESTED** [review]
See reply

File: src/main/java/Cell.java (hunk 1 above)
Comment (manuelkoso) [reply src/main/java/Cell.java:8]: Yes it can be

File: src/test/java/CellTests.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    void testGettersAfterCustomConstructor() {
+        Assertions.assertTrue(cell.isOccupied());
+        Assertions.assertEquals(cell.getStone(), stone);
+    }
+
+    @Test
+    void testSetStone() {
+        Stone newStone = new Stone(Color.WHITE);
+        cell.putStone(newStone);
+        Assertions.assertEquals(cell.getStone(), newStone);
```
Comment (peiva-git) [inline src/test/java/CellTests.java:26]: Should override equals method in Stone if using assertEquals

**Review by peiva-git — CHANGES_REQUESTED** [review]
Should override equals method in Stone if using assertEquals

File: src/test/java/CellTests.java (hunk 2 above)
Comment (manuelkoso) [reply src/test/java/CellTests.java:26]: You are right

**Comment by manuelkoso** [comment]
I suppose that two stones are equal whether they have the same color. I didn't consider the live field.

**Review by peiva-git — APPROVED** [review]
Great!

**Pull request merged**